frontend/catalyst/compiler.py

```diff
@@ -142,6 +142,7 @@ def run_writing_command(command: List[str], compile_options: Optional[CompileOpt
 QUANTUM_COMPILATION_PASS = (
     "QuantumCompilationPass",
     [
+        "specialize-active-callback-pass",
         "annotate-function",
         "lower-mitigation",
         "lower-gradients",
```

Comment:
Should this go directly before gradient lowering?

Reply:
Why?

Reply:
I think I was tying the callbacks to gradients because of the registration with enzyme. I was also thinking what is the lowest point in the pipeline it is first needed.

Is there a reason it should be the first thing in the pipeline?